### PR TITLE
Added Windows 10 to install/upgrade/uninstall

### PR DESCRIPTION
**Pull request by mkevinosullivan**
# WHY are these changes introduced?

Added references to Windows 10 to install/upgrade/uninstall documentation.

### WHAT is this pull request doing?

- Restructured install doc around systems first, then installation methods.
- Updated upgrade/uninstall docs to include references to Windows 10.
- Updated requirements section for Windows 10 on getting started landing page.

### CODE REVIEW

File: docs/getting-started/install/index.md [hunk 1]
```diff
@@ -16,23 +21,51 @@ $ brew tap shopify/shopify
 $ brew install shopify-cli
 ```
 
-### apt (Debian, Ubuntu)
+**RubyGems**
+
+See the RubyGems section below for further details.
```
Comment (paulomarg) [inline docs/getting-started/install/index.md:26]: Maybe we should add a link to the right section so users don't need to look for it themselves.

File: docs/getting-started/install/index.md [hunk 2]
```diff
@@ -16,23 +21,51 @@ $ brew tap shopify/shopify
...
+On Debian-based Linux systems, Shopify App CLI is available via the `apt` command _or_ RubyGems.
 
+**apt**
+
+You’ll need to install a downloaded `.deb` file with an explicit version number. Check the [releases page](https://github.com/Shopify/shopify-app-cli/releases) to make sure you install the latest package.
+
+1. Download the `.deb` file from the [releases page](https://github.com/Shopify/shopify-app-cli/releases)
+1. Install the downloaded file
 ```console
 $ sudo apt install shopify-cli-x.y.z.deb
```
Comment (paulomarg) [inline docs/getting-started/install/index.md:39]: We've had instances of people getting failures to do this by not using the right path to the `.deb` file. Should we make that explicit here?

File: docs/getting-started/install/index.md (hunk 1 above)
Comment (gfscott) [reply docs/getting-started/install/index.md:27]: +1!

File: docs/getting-started/install/index.md (hunk 2 above)
Comment (mkevinosullivan) [reply docs/getting-started/install/index.md]: I've added `/path/to/downloaded/shopify-cli-x.y.z...`

**Pull request merged**